templates/desktop/organisations.html

```diff
@@ -60,63 +60,63 @@ <h2>Countless apps available</h2>
     <div class="col-12">
       <ul class="p-matrix is-trisected">
         <li class="p-matrix__item">
-          <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2016/06/LibreOffice-Initial-Artwork-Logo.png" alt="">
+          <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2016/06/LibreOffice-Initial-Artwork-Logo.png?w=48" width="48" alt="">
```

Comment:
More dashboard.snapcraft.io links, which don't support the `?w=` param, in this file